staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -67,7 +67,15 @@ func NewDeltaFIFO(keyFunc KeyFunc, knownObjects KeyListerGetter) *DeltaFIFO {
 	return f
 }
 
-// DeltaFIFO is like FIFO, but allows you to process deletes.
+// DeltaFIFO is like FIFO, but allows the PopProcessFunc to process
+// deletes and adds Sync to the ways an object can be applied to an
```

Comment:
I have really hard time to parse this sentence (maybe it's because of my english, but I'm not sure what you're trying to say with it).

Reply:
reworded